src/components/utility/modal/index.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component, PropTypes} from 'react'
```

Comment:
You don't need the PropTypes import (or Component, if you make the export functional)